harvester_e2e_tests/apis/test_networks.py

```diff
@@ -99,3 +100,23 @@ def test_create_network_using_terraform(request, admin_session,
                                         harvester_api_endpoints,
                                         network_using_terraform):
     pass
+
+
+@pytest.mark.terraform
+@pytest.mark.public_network
+def test_cluster_network_using_terraform(request, admin_session,
+                                         harvester_api_endpoints):
+
+    vlan_nic = request.config.getoption('--vlan-nic')
+
+    network_data = utils.create_clusternetworks_terraform(
+                       request,
+                       admin_session,
+                       harvester_api_endpoints,
+                       'resource_clusternetworks',
+                       vlan_nic)
+
+    time.sleep(30)
+    assert vlan_nic in network_data['config']['defaultPhysicalNIC']
+    if not request.config.getoption('--do-not-cleanup'):
+        utils.destroy_resource(request, admin_session, 'dir-only')
```

Comment:
no need for all that code as you are already creating networking in the fixture

Reply:
I can remove the assert and --do-not-cleanup